python_modules/libraries/dagster-dbt/dagster_dbt/core/resource.py

```diff
@@ -783,7 +781,7 @@ def _get_subset_selection_for_context(
     # https://docs.getdbt.com/reference/node-selection/methods#the-path-method
     dbt_resource_props_by_output_name = get_dbt_resource_props_by_output_name(manifest)
     selected_dbt_non_test_resources = get_dbt_resource_names_for_output_names(
-        output_names=context.selected_output_names,
+        output_names=context.op_execution_context.selected_output_names,
```

Comment:
`AssetExecutionContext` raises a deprecation warning if `context.selected_output_names` is called, but allows you to get the underlying `op_execution_context`. `OpExecutionContext` also implements `op_execution_context`